app/models/reimbursement/report.rb

```diff
@@ -158,7 +160,11 @@ class Report < ApplicationRecord
       end
 
       event :mark_draft do
-        transitions from: [:submitted, :reimbursement_requested, :rejected], to: :draft
+        transitions from: [:submitted, :reimbursement_requested, :rejected, :reimbursement_approved], to: :draft
```

Comment:
hmmm, do we wanna drop this and require it to go from approved -> rejected -> draft?

i don't exactly remember how we can across the prod example in this weird state.

Reply:
Once it's approved, it shouldn't be able to go back to being a draft.